go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook
 
-go 1.18
+go 1.19
```

Comment:
Until we remove the 1.18 build, you'll need to revert this change. It's causing this issue in the CI:
```
++ make -j2 IMAGES=ceph build.all
go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18
```